src/components/_table.scss

```
@@ -263,3 +263,543 @@
     opacity: 1;
   }
 }
+
+// Table with scrollable content
+//
+// Markup change compare to not scrollable content :
+// - your table needs to be in a parent ("viewport") with a fixed size
+// - each column (th and td elements) are going to have the same width. But you can specify a width in percent for each column to adapt the grid.
+// - compare to previous one, on first th, don't forget to put "sb-table__checkbox" that define a specific size for the colum checkbox
+// - there is also a specific class for tbody (sb-table__tbody)
+//
+// Browser support: IE9+, Safari6+, Firefox30+, Chrome30+
+//
+// Markup:
+// <style>
+//   #kssref-components-table-scrollable td,
+//   #kssref-components-table-scrollable th {
+//     width: 10%;
+//   }
+//   #kssref-components-table-scrollable td:nth-child(1),
+//   #kssref-components-table-scrollable th:nth-child(1) {
+//     width: 40px;
+//   }
+//   #kssref-components-table-scrollable td:nth-child(2),
+//   #kssref-components-table-scrollable th:nth-child(2) {
+//     width: 50%;
+//   }
+// }
+// </style>
+// <div class="sb-table-container sb-overflow-x-auto">
+//   <table class="sb-table sb-table--scrollable-content {{modifier_class}}" role="grid">
```

```suggestion
//   <table class="sb-table__table" role="grid">
```
Related to comment below.